docs/Using-Fleet/REST-API.md

```diff
@@ -5170,6 +5172,14 @@ Returns a list of all queries in the Fleet instance.
     "name": "query1",
     "description": "query",
     "query": "SELECT * FROM osquery_info",
+    "team_id": null,
+    "frequency": 3600,
+    "platform": null,
```

Comment:
I thought this was "platforms"?

Rachael: We're not using plurals for this in the API yet.  It is just "platform" in the API.

Mike: My next thought is it would be `[]`, not `null`.

But turns out it is a comma-separated string.  What does the empty state look like for that string?  I would think it's "" but without testing the API manually I have no idea.  It would be very strange for it to be `null`.

Actually after reviewing the UI, it's clear there's no way to do an empty state on "platform" for policies.
<img width="465" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/31bd6962-f4ba-4410-9f52-10d97d432de5">

But in the schedule UI, there is a way for it to be empty.  
```
      "platform": "",
```

We tested it stays this way when "All" is selected and when "Select" is selected.

<img width="793" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/774edd1d-3e5b-45cf-9b16-b4e476371937">


This should be empty string, not `null`.